theia-full-docker/Dockerfile

```diff
@@ -187,6 +187,10 @@ RUN apt-get update \
 
 #.NET Core SDK
 ARG DOTNET_VERSION=3.1
+# disables .NET telemetry
+ENV DOTNET_CLI_TELEMETRY_OPTOUT 1
+# suppresses .NET welcome message
+ENV DOTNET_SKIP_FIRST_TIME_EXPERIENCE 1
```

Comment:
Good catch - it seems this entry should [speed-things up a bit for the image's build](http://www.donovanbrown.com/post/Stop-wasting-time-during-NET-Core-builds).